video2dataset/subsamplers/clipping_subsampler.py

```diff
@@ -58,7 +58,9 @@ def __call__(self, streams, metadata):
             clips = [clips]
 
         clips = [(s, e) for s, e in clips if get_seconds(e) - get_seconds(s) >= self.min_length]
-
+        if len(clips) == 0:
+            # return an error
+            return {}, [], "no viable clips"
```

Comment:
Lets give the user more info: f"Video had no clips with a length > {self.min_length}"